packages/cli/api-importers/openapi/openapi-ir-parser/src/openapi/v3/converters/convertToHttpError.ts

```diff
@@ -35,6 +35,7 @@ export const ERROR_NAMES_BY_STATUS_CODE: Record<number, string> = {
     449: "RetryWithError", // non-standard
     450: "BlockedByWindowsParentalControlsError", // non-standard
     451: "UnavailableForLegalReasonsError", // non-standard
+    498: "InvalidTokenError", // non-standard
```

Comment:
Not required here, but I imagine we might eventually need to support all the unofficial status codes listed [here](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) (since it looks like we've been adding them one by one).